zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java

```diff
@@ -210,6 +211,21 @@ public void testPersist() throws IOException, SchedulerException, RepositoryExce
   }
 
   @Test
+  public void testCreateNoteWithSubject() throws IOException, SchedulerException, RepositoryException {
+    AuthenticationInfo subject = new AuthenticationInfo("user1");
+    Note note = notebook.createNote(subject);
+
+    Notebook notebook = new Notebook(
```

Comment:
why creating new `notebook` instance here? can just reused previous one, isn't it